### PR TITLE
 Homework3: Ekaterina Tsiunel

### PR DESCRIPTION
**Pull request by ETsiunel**
homework-3-git-github change readme file

### CODE REVIEW

**Comment by Stml89** [comment]
Please, copy all *.yaml files(under .github directory) from my repository and turn on GitHub actions

**Comment by ETsiunel** [comment]
кажется done

ср, 1 мая 2024 г. в 20:06, Siarhei Stamal ***@***.***>:

File: .DS_Store
Comment (Stml89) [inline .DS_Store:1]: PLease, remove this file from the repository

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Stml89** [comment]
Also, remove 'github-actions-demo.yml' and 'main.yml' files from .github directory

**Comment by Stml89** [comment]
All workflow files should be under following directory -> `.github/workflows`
https://docs.github.com/en/actions/using-workflows/about-workflows#create-an-example-workflow

File: .DS_Store
Comment (ETsiunel) [reply .DS_Store:1]: done

**Comment by ETsiunel** [comment]
done: files removed, all files under `.github/workflows` directory

**Comment by Stml89** [comment]
Awesome, let's close this PR and open a new one, so GitHub actions will be triggered automatically.

I will improve actions soon, so you won't need to close\open PRs in the future

**Comment by ETsiunel** [comment]
I closed and then reopened - is it ok?

**Comment by Stml89** [comment]
Unfortunately not, close this one and open a new PR

**Pull request closed without merging**